.github/workflows/sycl_linux_build_and_test.yml

```diff
@@ -52,6 +52,11 @@ on:
         type: number
         required: false
         default: 4
+      uniq:
```

Comment:
This needs to be moved to Resolve Test Matrix job.

Reply:
Done